org.lflang/src/lib/ts/tsconfig.json

```diff
@@ -13,7 +13,7 @@
         "strictBindCallApply": true,
         "strictNullChecks": true,
         "strictFunctionTypes": true,
-        "typeRoots": ["./node_modules/@types/", "./node_modules/reactor-ts/src/core/@types/"]
+        "typeRoots": ["./node_modules/@types/", "./node_modules/@lf-lang/reactor-ts/src/core/@types/"]
```

Comment:
Note: the second element of the list should no longer be necessary once we remove our dependency on `ulog` as I suggested.